Equipment/EquipmentDb/Weapons/2hSwords.xml

```diff
@@ -27,6 +27,20 @@
 		</source>
 	</melee_weapon>
 
+	<melee_weapon id="13348" phase="1">
+		<info name="Demonshear" type="TWOHAND_SWORD" slot="2H" unique="no" req_lvl="58" item_lvl="63" quality="RARE" boe="no" icon="Inv_sword_14.png"/>
+		<dmg_range min="163" max="246" speed="3.8"/>
+		<proc>
+			<spell name="SHADOW_ATTACK" min="150" max="150" instant="yes" rate="0.06" internal_cd="0" spell_dmg_coefficient="0.0" duration="6" dmg_over_duration="120"/>
```

Comment:
Please remove the 'duration' and 'dmg_over_duration' attributes as they have no effect right now. We will add the support as a separate commit.